src/main/java/cli/Httpc.java

```diff
@@ -33,7 +37,17 @@ public Httpc() {
     @Command(description = "Executes an HTTP GET request for a given url")
     public void get(@Parameters(paramLabel = "URL") String url) {
         Map<String, String> parsedHeaders = parseHeaders(this.headers);
-        // TODO: send get request using TCPClient
+
+        try {
+            TCPClient tcpClient = new TCPClientImpl(url, 80);
+
+            HttpRequest getRequest = new HttpGetRequest("GET", tcpClient.getUri());
```

Comment:
I think it would make more sense to remove the http method argument from the constructor of `HttpGetRequest`, and instead call `super("GET", uri)`